sample_files/pre-commit-13.0/.editorconfig

```diff
@@ -0,0 +1,20 @@
+# Defaults for all kinds of source code
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# External web libs should be left untouched
+[*/static/lib/**]
```

Comment:
Which makes me think the exclude in `.pre-commit-config.yaml` is wrong (/static/*src/*lib). I'll fix it.

Reply:
Hm, actually we have /static/src/lib in some OCA modules too... should we exclude both?